.github/workflows/dockerImage.yml

```diff
@@ -45,10 +48,26 @@ jobs:
         uses: actions/download-artifact@v2
         with:
           name: targetfiles
-      - name: Prepare Docker variables
+      - name: Get current time
+        id: time
+        uses: nanzm/get-time-action@v1.1
+        with:
+          timeZone: 2
+          format: "YYYYMMDD[_]HHmmss"
+      - name: Prepare environment variables
         run: |
-          echo "DOCKER_REGISTRY=$(echo "docker.pkg.github.com/${{ github.repository }}" | awk '{print tolower($0)}')" >> $GITHUB_ENV
+          echo "DOCKER_REGISTRY=$(echo "${{ matrix.registry }}/${{ github.repository }}" | awk '{print tolower($0)}')" >> $GITHUB_ENV
```

Comment:
@niklasb99 , you forgot to define the matrix before. Please fix it.

Reply:
@koepferd done! 👍🏽